QUEST_LV_0100.tsv

```diff
@@ -8139,7 +8139,7 @@ QUEST_LV_0100_20150918_008138
 QUEST_LV_0100_20150918_008139	
 QUEST_LV_0100_20150918_008140	
 QUEST_LV_0100_20150918_008141	
-QUEST_LV_0100_20150918_008142	
+QUEST_LV_0100_20150918_008142	Use the condensed magic source on asleep Guardians
```

Comment:
If you want to use "asleep", the correct way is to reword the sentence -
"magic source on the Guardians who are asleep"
It may be a synonym, but you can't always use it in the exact same structure, just in the same way that you wouldn't say "the man who had fallen sleeping" to convey that he had fallen asleep, it would mean that he fell down while sleeping.